src/timesfm/__init__.py

```diff
@@ -11,7 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 """TimesFM init file."""
 
-from .timesfm import TimesFm, freq_map
+from timesfm.timesfm_base import freq_map, TimesFmCheckpoint, TimesFmHparams, TimesFmBase
+try:
```

Comment:
I am thinking what if one has both jax and torch version installed.

Reply:
Sg. Let's add 
print("Loaded Jax TimesFM.")
and
print("Loaded PyTorch TimesFM.")
after the imports respectively.

Reply:
Done